core-root/Home.tsx

```diff
@@ -89,76 +87,65 @@ const TODO = () => (
 )
 
 type FooterProps = {
-  pageIndex: number
   onPageIndexTapped: (index: number) => void
-  onCreateEventTapped: () => void
-  onProfileTapped: () => void
```

Comment:
So wait, why were these removed? Did we not need them anymore?

Reply:
`useCoreNavigation`